src/app/util/helpers.js

```
@@ -4,5 +4,26 @@ module.exports = {
         let parts = str.split('.');
         parts[0] = parts[0].replace(/\B(?=(\d{3})+(?!\d))/g, ',');
         return parts.join('.');
+    },
+    validatePhone: function (phoneNo) {
+        if (!phoneNo || phoneNo.trim().length <= 0) {
+            return "Phone number is required";
+        }
+        if (isNaN(phoneNo)) {
+            return "Phone must be a number";
+        }
+        if (phoneNo.length !== 10) {
+            return "Phone number must be 10 digits";
+        }
+        return "";
+    },
+    validateAmount: function (amount) {
```

We should compare the balance and the amount too. If the balance only has 100 but the amount is 101, we should throw an error
The amount field should have a min value like 0 or 1000

---

@leonthaibao It will add balance check case soon

---

@leonthaibao The minimum amount that can be transferred is 1 VND